src/game/client/worldheightmap.cpp

```diff
@@ -693,7 +693,7 @@ bool WorldHeightMap::Parse_Blend_Tile_Data(DataChunkInput &file, DataChunkInfo *
         m_textureClasses[i].first_tile = file.Read_Int32();
         m_textureClasses[i].num_tiles = file.Read_Int32();
         m_textureClasses[i].width = file.Read_Int32();
-        file.Read_Int32();
+        m_textureClasses[i].is_blend_edge_tile = file.Read_Int32();

```

Comment:
not too sure if this should be accepted, don't know the implications of fixing this..

Reply:
@tomsons26 isn't point of restoring game sources in modifying them and fixing developer errors, as well as improving support for modern systems?

Reply:
it is eventually, but we need to know what impact it has on the game